mlrun/feature_store/steps.py

```diff
@@ -70,6 +70,9 @@ def _do_storey(self, event):
     def _do_spark(self, event):
         raise NotImplementedError
 
+    def validate(self, feature_set):
```

Comment:
instead of having a default validate(), remove this and then in valiadate_steps() check if the class has validate() method before going ahead and creating the object.

Reply:
fixed